core/backend/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -958,4 +959,177 @@ describe("IModelTransformer", () => {
     targetDb.close();
   });
 
+  it("processSchemas should wait for the schema import to finish to delete the export directory", async () => {

```

Comment:
the diff looks weird, seemingly because the master branch on your fork is out of date. Could you fix that before we merge, just so the merge is clearer and we don't let anything slip in? This `processSchemas should wait for the schema import to finish to delete the export directory` test is already in master on the main remote, but appears in this diff.

Reply:
yeah I had some trouble syncing my fork with this repository, hopefully did it right this time